src/lib/locale/index.ts

```diff
@@ -119,6 +119,14 @@ const routing = defineRouting({
       en: '/shift-schedule',
       no: '/vaktliste',
     },
+    '/reservations': {
+      en: '/reservations',
+      no: '/reservasjoner',
+    },
+    '/reservations/[tool]': {
```

Comment:
```suggestion
    '/reservations/[toolId]': {
```

Use Id in the name as that is more explicit